pkg/rpc/server/error.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/nspcc-dev/neo-go/pkg/rpc/response"
+)
+
+// serverError represents RPC error response on the server side.
+type serverError struct {
+	*response.Error
+	HTTPCode int // HTTPCode won't be marshalled because Error's marshaller is used.
```

Comment:
Can we use `Abstract` for that? Set 200 by default and then transform `packClientError` into `getHTTPCodeForError`? This will allow us to avoid `serverError` completely.